nullaway/src/main/java/com/uber/nullaway/handlers/Handler.java

```diff
@@ -182,16 +181,17 @@ Nullness onOverrideMethodInvocationReturnNullability(
    * @param isAnnotated A boolean flag indicating whether the called method is considered to be
    *     within isAnnotated or unannotated code, used to avoid querying for this information
    *     multiple times within the same handler chain.
-   * @param argumentPositionNullness The argument position to nullness map computed by upstream
-   *     handlers and/or the base analysis (though information from the base analysis is allowed to
-   *     be incomplete/sparse at this point).
-   * @return The updated argument position to nullness map, as computed by the current handler.
+   * @param argumentPositionNullness Nullness info for each argument position as computed by
+   *     upstream handlers and/or the base analysis. Some entries may be {@code null}, indicating
+   *     upstream analysis and handlers model the parameter as unannotated.
```

Comment:
I wonder if "as unannotated" is the right term once you mention handlers... it's definitely possible for handlers to have a more specific opinion on unannotated code and thus an explicit value here (e.g. `RestrictiveAnnotationHandler` will explicitly mark `@Nullable` parameters within unannotated code as `NULLABLE` here). I guess "model the parameter as nullness-unknown, which usually indicates parameters within unannotated code for which no handler specifies any particular nullability"?

Reply:
Sure, I will clarify